pkg/controller/jobframework/defaults.go

```diff
@@ -96,6 +100,39 @@ func ApplyDefaultLocalQueue(jobObj client.Object, defaultQueueExist func(string)
 	}
 }
 
+func CopyLabelAndAnnotationFromOwner(ctx context.Context, jobObj client.Object, k8sClient client.Client) {
```

Comment:
This is very specific function, but the name is generic.
Let's have something like `RaySubmitterJobCopyLabelAndAnnotationFromOwner`.